README.md

```diff
@@ -49,6 +49,42 @@ Run `sh build.sh help` to see all the actions you can perform (build ios/osx, ge
 
 Executing the examples under the `examples/` folder, requires that you have built the `Realm.framework`.
 
+
+## Branches
+
+Here is an overview of some long-running branches we're currently using in development, which you may wish to use.
+
+_:warning: Feedback is very welcome for unreleased version, but please aware that we can't provide support to the same
```

Comment:
It was more about making sure to direct the feedback directly to the PR and making aware to avoid shipping a prerelease to the AppStore.